Dockerfile

```
@@ -1,29 +1,29 @@
-FROM openshift/origin-cli:v3.11 AS builder
-
-RUN cp /bin/oc /
+FROM openshift/origin-cli:4.12 AS builder
```

Should we be explicit with using quay here (otherwise it may default to Docker Hub)?

```suggestion
FROM quay.io/openshift/origin-cli:4.12 AS builder
```

---

These are using imagestreams in the build cluster